src/components/admin/Button/Button.tsx

```diff
@@ -5,7 +5,11 @@ import { Loading } from "components/molecules/Loading";
 
 import * as TW from "./Button.tailwind";
 
-export type ButtonVariant = "primary" | "secondary" | "danger";
+export type ButtonVariant =
+  | "primary"
+  | "secondary"
+  | "secondaryBorderless"
```

Comment:
This might be better if moved as `| borderless` for `ButtonBorders` as discussed above in the `.tailwind.ts` file